test-utils/src/rpc/data/json.rs

```diff
@@ -0,0 +1,1292 @@
+    "seed_hash": "d432f499205150873b2572b5f033c9c6e4b7c6f3394bd2dd93822cd7085e7307"
+  }
+}"#;
+    Response =
+r#"{
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": "d0402d6834e26fb94a9ce38c6424d27d2069896a9b8b1ce685d79936bca6e0a8"
+}"#;
+}
+
+define_request_and_response! {
+    flush_cache (json_rpc),
+    FLUSH_CACHE: &str,
+    Request =
+r#"{
+  "jsonrpc": "2.0",
+  "id": "0",
+  "method": "flush_cache",
+  "params": {
+    "bad_txs": true,
+    "bad_blocks": true
+  }
+}"#;
+    Response =
+r#"{
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "status": "OK",
+    "untrusted": false
+  }
+}"#;
+}
+
+define_request_and_response! {
+    add_aux_pow (json_rpc),
+    ADD_AUX_POW: &str,
+    Request =
+r#"{
+  "jsonrpc": "2.0",
+  "id": "0",
+  "method": "add_aux_pow",
+  "params": {
+    "blocktemplate_blob": "1010f4bae0b4069d648e741d85ca0e7acb4501f051b27e9b107d3cd7a3f03aa7f776089117c81a0000000002c681c30101ff8a81c3010180e0a596bb11033b7eedf47baf878f3490cb20b696079c34bd017fe59b0d070e74d73ffabc4bb0e05f011decb630f3148d0163b3bd39690dde4078e4cfb69fecf020d6278a27bad10c58023c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    "aux_pow": [{
+      "id": "3200b4ea97c3b2081cd4190b58e49572b2319fed00d030ad51809dff06b5d8c8",
+      "hash": "7b35762de164b20885e15dbe656b1138db06bb402fa1796f5765a23933d8859a"
+    }]
+  }
+}"#;
+    Response =
+r#"{
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "aux_pow": [{
+      "hash": "7b35762de164b20885e15dbe656b1138db06bb402fa1796f5765a23933d8859a",
+      "id": "3200b4ea97c3b2081cd4190b58e49572b2319fed00d030ad51809dff06b5d8c8"
+    }],
+    "blockhashing_blob": "1010ee97e2a106e9f8ebe8887e5b609949ac8ea6143e560ed13552b110cb009b21f0cfca1eaccf00000000b2685c1283a646bc9020c758daa443be145b7370ce5a6efacb3e614117032e2c22",
+    "blocktemplate_blob": "1010f4bae0b4069d648e741d85ca0e7acb4501f051b27e9b107d3cd7a3f03aa7f776089117c81a0000000002c681c30101ff8a81c3010180e0a596bb11033b7eedf47baf878f3490cb20b696079c34bd017fe59b0d070e74d73ffabc4bb0e05f011decb630f3148d0163b3bd39690dde4078e4cfb69fecf020d6278a27bad10c58023c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    "merkle_root": "7b35762de164b20885e15dbe656b1138db06bb402fa1796f5765a23933d8859a",
+    "merkle_tree_depth": 0,
+    "status": "OK",
+    "untrusted": false
+  }
+}"#;
+}
+
+define_request_and_response! {
+    UNDOCUMENTED_METHOD (json_rpc),
+    GET_TX_IDS_LOOSE: &str,
+    Request =
+r#"{
+  "jsonrpc": "2.0",
+  "id": "0",
+  "method": "get_txids_loose",
+  "params": {
+    "txid_template": "0000000000000000aea473c43708aa50b2c9eaf0e441aa209afc9b43458fb09e",
+    "num_matching_bits": 192
+  }
+}"#;
+    Response =
+r#"{
+  "id": "0",
+  "jsonrpc": "2.0",
+  "result": {
+    "txids": "",
+    "status": "OK",
+    "untrusted": false
+  }
+}"#;
+}
```

Comment:
TODO: this has to be moved in `rpc` as well, it should be in `json.rs` not `other.rs`.